static/app/views/performance/browser/resources/resourceSummaryPage/sampleImages.tsx

```diff
@@ -128,6 +132,22 @@ function ImageContainer(props: {
   );
 }
 
+function useGeneralProjectSettings(projectId: number) {
+  const api = useApi();
+  const {projects} = useProjects();
+  const stringProjectId = projectId.toString();
+  const project = projects.find(p => p.id === stringProjectId);
+  const organization = useOrganization();
```

Comment:
l: Let's move this hook to below line 137 so that we don't accidentally cause conditional hooks in future refactors.

Reply:
👍